crates/playground/README.md

```diff
@@ -0,0 +1,69 @@
+<div align="center">
```

Comment:
I think we can drop the content of the README because it's just a template text.

Reply:
Replaced by symbolic link to parent README